### PR TITLE
feat: replace env private keys with keystores

### PR DESCRIPTION
**Pull request by 0xJabberwock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,13 @@ yarn coverage
 
 ### Setup
 
-Configure the `.env` variables.
+```bash
+cast wallet import MAINNET_DEPLOYER --interactive
```
Comment (gas1cent) [inline README.md:108]: What do you think about giving devs the choice to name the wallet the way they want? We could do it through the `.env` file.

**Review by gas1cent — DISMISSED** [review]

**Review by 0xShaito — DISMISSED** [review]

File: .env.example [hunk 2]
```diff
@@ -1,7 +1,7 @@
 MAINNET_RPC=
-MAINNET_DEPLOYER_PK=
+MAINNET_DEPLOYER=
```
Comment (wei3erHase) [inline .env.example:2]: wdyt? i think isn't clear enough :/
```suggestion
MAINNET_DEPLOYER_ADDRESS=
```

**Review by wei3erHase — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (wei3erHase) [reply README.md]: hmmm seems unnecessary to build an .env to name the deployer, isn't foundry storing it locally? 
is "forcing a dev to choose a name" (setting the env), vs "just using `MAINNET_DEPLOYER`" and allowing the dev to change the package.json

File: .env.example (hunk 2 above)
Comment (wei3erHase) [reply .env.example:2]: got it wrong, but perhaps we should use a `_ID` or `_NAME` in the .env, or explain what are we doing on the readme 🙏

File: README.md (hunk 1 above)
Comment (0xJabberwock) [reply README.md]: Changing the `package.json` would require every dev to declare (or redeclare) such account locally in order to be able to run `yarn deploy`. However, I agree with that forcing a dev to choose a name is generally undesired. Maybe we could update `MAINNET_DEPLOYER` at the very beginning of each project, setting it to a more ad hoc account name.

**Review by wei3erHase — APPROVED** [review]
😘🤌🏻

**Review by gas1cent — APPROVED** [review]

**Review by wei3erHase — APPROVED** [review]
👍

**Review by gas1cent — APPROVED** [review]
😍

**Review by 0xShaito — APPROVED** [review]

**Pull request merged**